### ISSUE TITLE
[iOS] static lib is missing loop target

### ISSUE THREAD

**Issue by RomainQuidet** [mapbox/mapbox-gl-native#4714]
Platform: iOS
Mapbox SDK version: custom build on master, commit iOS 3.2.0

### Steps to trigger behavior

 1. run make ipackage
 2. get the static framework
 3. link with an app

### Expected behavior
app build success

### Actual behavior
app build fails, missing som mapbox util classes.

=> in package.sh, the static lib is build with aggregation of multiple targets instead of target everything, but it's missing loop target
Bug introduced by https://github.com/mapbox/mapbox-gl-native/commit/08d12f860e3eeac810cbb9355eb168fc8de38ce2#diff-86a1df829aa488a1c4289ab852eb2961

**Comment by 1ec5** [comment]
This code is being rewritten in #4641.

/cc @jfirebaugh

**Comment by jfirebaugh** [comment]
Can you please post the build log showing the error?

**Comment by jfirebaugh** [comment]
Also what commit are you building off of? I think this was already fixed by f964e40e7e9220d08751d8607af61ac5a7c0794c.

**Comment by 1ec5** [comment]
Closing. Please open a new issue if you still see build breakage after #4641.
